### PR TITLE
Create Exactly-About.info.xml

### PR DESCRIPTION
**Pull request by MojtabaFaal**

### CODE REVIEW

File: src/chrome/content/rules/Exactly-About.info.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<ruleset name="Exactly-About.info">
+        <target host="exactly-about.info" />
+        <target host="www.exactly-about.info" />
+		
+		<rule from="^http://(?:www\.)?exactly-about\info/" to="https://exactly-about.info/"/>
```
Comment (jsha) [inline src/chrome/content/rules/Exactly-About.info.xml]: This line is redundant with the one below it, please delete it.

Comment (MojtabaFaal) [inline src/chrome/content/rules/Exactly-About.info.xml]: We talked to each other
And you said that I remove the line 7 and 8
Thank you

Comment (RDewiAR27) [inline src/chrome/content/rules/Exactly-About.info.xml]: Ud malam berisik dongoo

**Comment by jsha** [comment]
One more fix: `exactly-about\info` is incorrect, it should be `exactly-about\.info`.

**Comment by jsha** [comment]
Oops, did you mean to close this?

**Comment by MojtabaFaal** [comment]
After the changes, the error was generated
" Failure: The Travis CI build failed "

**Comment by MojtabaFaal** [comment]
The new "Pull request" is also error
" Failure: The Travis CI build failed "
https://github.com/EFForg/https-everywhere/pull/2287
why ?

**Comment by jsha** [comment]
See my comment above: you are missing a `.`. Line 5 should say

```
<rule from="^http://(?:www\.)?exactly-about\.info/" to="https://exactly-about.info/"/>
```

**Comment by MojtabaFaal** [comment]
So why in the first, was right?
https://github.com/EFForg/https-everywhere/commit/5e09ad1385a2ee19f4d5a8f931fc1d6e45c50a80

**Comment by jsha** [comment]
The tests check that at least one rule matches each target host. In the first version you posted, there was a second rule that did match each target host, so the tests passed. In other words, the second rule hid the fact that the first rule was buggy. Now that there is only one rule, the tests can find the problem!

**Comment by MojtabaFaal** [comment]
Thanks
I realized

**Comment by MojtabaFaal** [comment]
All checks have passed
https://github.com/EFForg/https-everywhere/pull/2287

**Comment by MojtabaFaal** [comment]
If true, please paste a ==> Label <==
https://github.com/EFForg/https-everywhere/pull/2287
Thanks

**Pull request closed without merging**